Lib/asyncio/events.py

```diff
@@ -21,6 +23,7 @@
 import subprocess
 import sys
 import threading
+import warnings
```

Comment:
We generally use local imports for deprecation warnings. WDYT?

Reply:
The upcoming PRs will add more warnings so I kept it at module level